numpy/_core/tests/test_umath.py

```diff
@@ -422,13 +422,14 @@ def test_unsigned_signed_direct_comparison(
 
         assert py_comp(arr, -1) == expected
         assert np_comp(arr, -1) == expected
+
+
         scalar = arr[0]
         assert isinstance(scalar, np.integer)
         # The Python operator here is mainly interesting:
         assert py_comp(scalar, -1) == expected
         assert np_comp(scalar, -1) == expected
 
-
```

Comment:
Two empty lines between classes is good.

Reply:
strange, probably some conflict resolution